lib/db.js

```diff
@@ -334,7 +334,7 @@ class LimitDBRedis extends EventEmitter {
             conformant,
             remaining,
             reset: Math.ceil(reset / 1000),
-            limit: bucketKeyConfig.size,
+            limit: erl_activated ? elevated_limits.size : bucketKeyConfig.size,
```

Comment:
just want to check something; `erl_activated` is only `true` when the bucket being used is configured for erl, not just when erl is active for the tenant, correct?

if so, great. if not, we could have issues here as it would try to display an erl size for a bucket without an erl config.

Reply:
@LeweyM is right. It'll return activated if the key exists. I'll work on that

Reply:
Offlined: Agreed we'll merge it and follow-up with the above mentioned fix in the target branch.